MahApps.Metro/Controls/Dialogs/LoginDialog.cs

```diff
@@ -36,12 +40,17 @@ public LoginDialogSettings()
         public Visibility NegativeButtonVisibility { get; set; }
 
         public bool EnablePasswordPreview { get; set; }
+
+        public Visibility RememberCheckBoxVisibility { get; set; }
+
+        public string RememberCheckBoxText { get; set; }
     }
 
     public class LoginDialogData
     {
         public string Username { get; set; }
         public string Password { get; set; }
+        public bool Remember { get; set; }
```

Comment:
I would change the name of this property to a more `boolean-sounding` name. I would suggest something like *IsRememberRequested*, *RequestRemember*. This is because the developer might be confused because it might understand that MahApps does remember the state of the dialog.
Another possibility would be to add a documentation tag to the property.